examples/react/contexts/WalletSelectorContext.tsx

```diff
@@ -12,6 +12,7 @@ import { setupNearFi } from "@near-wallet-selector/nearfi";
 import { setupNightly } from "@near-wallet-selector/nightly";
 import { setupNightlyConnect } from "@near-wallet-selector/nightly-connect";
 import { setupSender } from "@near-wallet-selector/sender";
+import { setupBitgetWallet } from "@near-wallet-selector/bitget-wallet/src";
```

Comment:
This import is wrong, can you remove the `/src` part?

Can you add the setup on these places too:
https://github.com/near/wallet-selector/blob/main/examples/react/contexts/WalletSelectorExportContext.tsx

https://github.com/near/wallet-selector/blob/main/examples/angular/src/app/pages/wallet-selector/wallet-selector.component.ts

https://github.com/near/wallet-selector/blob/main/examples/angular/src/app/pages/wallet-selector-export/wallet-selector-export.component.ts